packages/components/organisms/f-header/src/components/NavLink.vue

```diff
@@ -1,22 +1,25 @@
 <template>
-    <div>
+    <a
+        v-bind="$attrs"
+        :data-trak="JSON.stringify(dataTrak)"
+        :href="href"
+        :class="[
+            { [$style['c-navLink']]: !isCountryLink },
+            { [$style['c-navLink--countryLink']]: isCountryLink },
+            { [$style['c-nav-list-link--alt']]: isAltColour },
+            { [$style['c-navLink--whiteBackground']]: backgroundTheme === 'white' },
+            { [$style['c-nav-list-link--transparent']]: backgroundTheme === 'transparent' },
+            { [$style['c-navLink--countryLink']]: isCountryLink }
+        ]">
```

Comment:
Minor thing, but you should be able to remove a lot of the curly braces here:

```suggestion
        :class="[{
            [$style['c-navLink']]: !isCountryLink,
            [$style['c-navLink--countryLink']]: isCountryLink,
            [$style['c-nav-list-link--alt']]: isAltColour,
            [$style['c-navLink--whiteBackground']]: backgroundTheme === 'white',
            [$style['c-nav-list-link--transparent']]: backgroundTheme === 'transparent',
            [$style['c-navLink--countryLink']]: isCountryLink
        }]">
```

Reply:
awesome, thank you! just changed 👍